client/scripts/directives/sidebar.js

```diff
@@ -2,9 +2,20 @@ require('../app').directive('sidebar', /* @ngInject */function () {
   return {
     templateUrl: '/views/directives/sidebar.html',
     scope: {},
-    controller: /* @ngInject */function ($scope, $q, $state, api) {
-      this.isPrivateCollapsed = !$state.includes('auth.monitoring.private')
-      this.isPublicCollapsed = !$state.includes('auth.monitoring.public')
+    controller: /* @ngInject */function ($scope, $rootScope, $q, $state, api) {
+      var ctrl = this
+
+      function updateFlags () {
+        ctrl.isPrivateCollapsed = !$state.includes('auth.monitoring.private')
+        ctrl.isPublicCollapsed = !$state.includes('auth.monitoring.public')
+        ctrl.isStatsCollapsed = !$state.includes('auth.stats')
+      }
+
+      updateFlags()
+
+      $rootScope.$on('$stateChangeSuccess', function () {
```

Comment:
this is leaving dangling event handler

```suggestion
      $scope.$on('$destroy', $rootScope.$on('$stateChangeSuccess', function () {
```